project.clj

```diff
@@ -17,71 +17,80 @@
+                           [nginx-clojure                   "0.4.4"]]
+          :plugins
+                          [;;; These must be in :dev, Ref. https://github.com/lynaghk/cljx/issues/47:
+                           [com.keminglabs/cljx             "0.6.0"
+                            :exclusions [org.clojure/clojure com.cemerick/piggieback]]
+                           [lein-cljsbuild                  "1.1.3"]
+                           [lein-codox                      "0.9.4"
+                            :exclusions [org.clojure/clojure]]]
+          :source-paths ["src" "target/generated/src/clj" "test"]
+          :resource-paths ["target/generated/src/cljs"]
+          :test-paths   ["src" "target/generated/src/clj" "test"]
+          :figwheel {:ring-handler taoensso.sente.test-server/relay-app}
+          :clean-targets ^{:protect false} [:target-path "dev-resources"]
+          :cljsbuild {:builds
+                      [{:id           "devcards"
+                        :source-paths ["src" "test" "target/generated/src/cljs"]
+                        :figwheel     {:devcards true}
+                        :compiler     {:main                 "taoensso.sente.devcards"
+                                       :asset-path           "js/compiled/devcards_out"
+                                       :output-to            "dev-resources/public/js/compiled/sente_devcards.js"
+                                       :output-dir           "dev-resources/public/js/compiled/devcards_out"
+                                       :source-map-timestamp true}}]}}]}
 
-  :test-paths ["test" "src"]
   :prep-tasks [["cljx" "once"] "javac" "compile"]
   :codox
   {:language :clojure ; [:clojure :clojurescript] ; No support?
-   :source-paths ["target/classes"]
+   :source-paths ["target/generated/src/clj"]
    :source-uri
    {#"target/classes"
-    "https://github.com/ptaoussanis/sente/blob/master/src/{classpath}x#L{line}"
+    "https://github.com/ptaoussanis/sente/blob/master/src/{classpath}#L{line}"
     #".*"
     "https://github.com/ptaoussanis/sente/blob/master/{filepath}#L{line}"}}
 
   :aliases
-  {"test-all"   ["do" "clean," "cljx" "once,"
-                 "with-profile" "+1.6:+1.7:+1.8" "test,"
-                 ;; "with-profile" "+test" "cljsbuild" "test"
-                 ]
-   "build-once" ["do" "cljx" "once," "cljsbuild" "once"]
+  {"build-once" ["do" "cljx" "once," "cljsbuild" "once"]
+   "test-all"   ["do"
+                 "clean,"
+                 "build-once,"
+                 "test,"]
+   "devcards"   ["do"
+                 "clean,"
+                 "cljx" "once,"
+                 "figwheel,"]
    "deploy-lib" ["do" "build-once," "deploy" "clojars," "install"]
    "start-dev"  ["with-profile" "+server-jvm" "repl" ":headless"]}
 
   :repositories
-  {"sonatype-oss-public" "https://oss.sonatype.org/content/groups/public/"})
+  {"sonatype-oss-public" "https://oss.sonatype.org/content/groups/public/"})
```

Comment:
Add a newline at the end of this file (probably something you can configure your editor to do).